modules/ossm-threescale-webassembly-module-api-object.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assembly:
+//
+// service_mesh/v2x/ossm-threescale-webassembly-module.adoc
+
+[id="ossm-threescale-webassembly-module-api-object_{context}"]
+= The 3scale WebAssembly module `api` object
+
+The `api` top-level string from the 3scale WebAssembly module defines which version of the configuration the module will use.
+
+NOTE: A non-existent or unsupported version of the `api` object renders the 3scale WebAssembly module inoperable.
+
+.The `api` top level string example
+----
+spec:
+  config:
+    api: v1
+----
```

Comment:
```suggestion
.The `api` top level string example
[source,yaml]
----
spec:
  config:
    api: v1
...
----
```

I'd add `...` to show this is an excerpt, and also I'd add in the metadata at the start of the resource if possible so you can see some context, e.g.

```
apiVersion: maistra.io/v1alpha1
kind: ServiceMeshExtension
metadata:
  name: threescale-wasm-auth
  namespace: bookinfo
spec:
  config: v1
...
```
it seems like the 3scale WebAssembly module might be a custom resource but there's no example of one here?